quickevent/app/plugins/qml/Event/src/Event/eventconfig.cpp

```diff
@@ -106,13 +106,13 @@ void EventConfig::save(const QString &path_to_save)
 			val_str = val.toString();
 		q_up.bindValue(":key", key);
 		q_up.bindValue(":val", val);
-		q_up.exec();
+		q_up.exec(qf::core::Exception::Throw);
```

Comment:
Understood.  A few lines above there is
```
q_up.prepare("UPDATE config SET cvalue=:val WHERE ckey=:key", qf::core::Exception::Throw);
...
q_ins.prepare("INSERT INTO config (ckey, cvalue, ctype) VALUES (:key, :val, :type)", qf::core::Exception::Throw);
```
Should these occurrences of exception throwing be replaced with return value checking too?